### PR TITLE
Implements the `call-webhook` action and adds all Action client-side schemata

### PR DESCRIPTION
**Pull request by chrisguidry**
Adds the `CallWebhook` action to use a invoke a configured Webhook block. This
also adds client-side schemata for all supported Action types on both OSS
and Prefect Cloud.

### CODE REVIEW

File: src/prefect/server/utilities/http.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    if key.startswith("x-forwarded"):
+        return True
+    if key.startswith("x-b3"):
+        return True
+    if key.startswith("x-cloud-trace"):
+        return True
+    if key.startswith("x-envoy"):
+        return True
+    if key.startswith("x-nebula"):
+        return True
```
Comment (zzstoatzz) [inline src/prefect/server/utilities/http.py:16]: is this pertinent for OSS?

**Review by zzstoatzz — APPROVED** [review]
lgtm! (schemata let's gooo)

File: src/prefect/server/utilities/http.py (hunk 1 above)
Comment (chrisguidry) [reply src/prefect/server/utilities/http.py:16]: Ohh good point, this probably is not, will remove

File: src/prefect/events/actions.py [hunk 2]
```diff
@@ -1,63 +1,290 @@
...
+class ResumeAutomation(AutomationAction):
+    """Resumes a Work Queue"""
+
+    type: Literal["resume-automation"] = "resume-automation"
+
+
+class DeclareIncident(Action):
+    """Declares an incident for the triggering event.  Only available on Prefect Cloud"""
+
+    type: Literal["declare-incident"] = "declare-incident"
```
Comment (desertaxle) [inline src/prefect/events/actions.py:266]: This action likely isn't needed?

**Pull request merged**